omniduct/databases/base.py

```diff
@@ -34,29 +34,12 @@ def cache_deserializer(format):
 
 @decorator
 def render_statement(method, self, statement, *args, **kwargs):
-    # Check if statement is an SQLAlchemy executable expression, and if so, render it
-    try:
-        from sqlalchemy.sql.base import Executable
-        if isinstance(statement, Executable):
-            statement = str(statement.compile(compile_kwargs={"literal_binds": True}))
-    except ImportError:
-        pass
-
-    # If templating enabled, render template
-    if kwargs.pop('template', False):
-        template_context = {}
-        template_context.update(self._template_context)
-        kwarg_context = kwargs.pop('template_context', {})
-        template_context.update(kwarg_context)
-        intersection = set(self._template_context.keys()) & set(kwarg_context.keys())
-        if intersection:
-            logger.warning(
-                "The following default template context keys have been overridden "
-                "by the local context: {}."
-                .format(intersection)
-            )
-        statement = self.render_template(statement, template_context)
-
+    statement = self.render_template(
```

Comment:
Why not put this inside a switch, so we don't need to add a "template" kwarg to `render_template`?

```
if kwargs.pop('template', True):
    statement = self.render_template(...)
```